rust/hyper/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "server"
+version = "0.0.0"
+edition = "2021"
+
+[dependencies]
+http-body-util = "*"
```

Comment:
this package is in dev-dependencies in https://github.com/hyperium/hyper-util/blob/5f055e36d6f945b51b4d4b023290f551ae7eb4b6/Cargo.toml#L36. any idea why ?

Reply:
I really don't think that using `*` as a dependency version is ever a good, future-proof solution. If this is really what we want to do, I recommend you add a comment that explains why exactly this is a good idea here

Reply:
Regarding why it is in the dev dependencies - because they use it in their example code, see https://github.com/search?q=repo%3Ahyperium%2Fhyper-util%20http_body_util&type=code